dmcontent/questions.py

```diff
@@ -464,6 +467,9 @@ def get_question(self, field_name):
         if self.id == field_name or field_name in self.fields.values():
             return self
 
+    def unformat_data(self, data):
+        return self.get_data(data)
```

Comment:
I think this part is interesting. These 2 lines are confusing because `unformat_data` and `get_data` perform opposite operations, so they can't have the same implementation.

The reason we need this with the proposed implementation of `ContentSection.unformat_data` is that it delegates based on `question.get_question` and `Pricing` returns itself whenever any of it's fields are passed in. Which means that if I understand this correctly this method will get called N times for a pricing question with N fields and each time the return value will overwrite all pricing fields in the section's unformatted data dictionary. Which is not a problem because this return value is always the same.

However, this seems to suggest that we might simplify this by passing in the original field `key` to the `quesiton.unformat_data(self, data, key)` - which then allows us to use the base class implementation and return `{key: data[key]}`.